diff --git a/node_modules/@beyonk/svelte-datepicker/src/components/lib/date-utils.js b/node_modules/@beyonk/svelte-datepicker/src/components/lib/date-utils.js
index de41297..06a53d1 100644
--- a/node_modules/@beyonk/svelte-datepicker/src/components/lib/date-utils.js
+++ b/node_modules/@beyonk/svelte-datepicker/src/components/lib/date-utils.js
@@ -3,11 +3,13 @@ import localeData from 'dayjs/plugin/localeData'
 import minMax from 'dayjs/plugin/minMax'
 import isSameOrBefore from 'dayjs/plugin/isSameOrBefore'
 import isSameOrAfter from 'dayjs/plugin/isSameOrAfter'
+import de from 'dayjs/locale/de'
 
 dayjs.extend(localeData)
 dayjs.extend(minMax)
 dayjs.extend(isSameOrBefore)
 dayjs.extend(isSameOrAfter)
+dayjs.locale(de)
 
 export {
   dayjs
diff --git a/node_modules/@beyonk/svelte-datepicker/src/components/view/date-view/Month.svelte b/node_modules/@beyonk/svelte-datepicker/src/components/view/date-view/Month.svelte
index d02379b..5136d99 100644
--- a/node_modules/@beyonk/svelte-datepicker/src/components/view/date-view/Month.svelte
+++ b/node_modules/@beyonk/svelte-datepicker/src/components/view/date-view/Month.svelte
@@ -21,7 +21,7 @@
   <div class="month-dates">
     <div class="legend">
       <div class="month-week">
-        {#each dayjs.weekdaysShort() as day}
+        {#each dayjs.weekdaysShort(true) as day}
           <span>{day}</span>
         {/each}
       </div>
